chartfx-chart/pom.xml

```diff
@@ -13,6 +13,9 @@
     <name>chartfx-chart</name>
     <properties>
         <project.moduleName>io.fair-acc.chartfx</project.moduleName>
+        <sass.version>1.64.2</sass.version>
+        <scss.inputDir>${project.basedir}/src/main/resources/io/fair_acc/chartfx/</scss.inputDir>
+        <css.outputDir>${scss.inputDir}</css.outputDir>
```

Comment:
``` diff
-  <scss.inputDir>${project.basedir}/src/main/resources/io/fair_acc/chartfx/</scss.inputDir>
- <css.outputDir>${scss.inputDir}</css.outputDir>
+ <scss.inputDir>${project.basedir}/src/main/scss/</scss.inputDir>
+ <scss.outputDir>${project.basedir}/src/main/resources/io/fair_acc/chartfx/</scss.outputDir>
```
If` I'm not misstaken this leads to both the scss and css files being copied into the resulting jar, while only the generated css files are required. Not a big deal, but it might be cleaner to have them in a separate directory. As I don't have a lot of experience with scss I'm not sure how this all plays together.

Also the idea is to have scss and the generated css files in the git repo?